nad/ITRF2008

```diff
@@ -32,30 +32,30 @@
 # J. Geophys. Res., 117, B07402, doi:10.1029/2011JB008930.
 
 
-<AMUR> +proj=helmert +drx=-0.000190 +dry=-0.000442 +drz=0.000915
+<AMUR> +proj=helmert +drx=-0.000190 +dry=-0.000442 +drz=0.000915 +convention=coordinate_frame
```

Comment:
Confirmed. Position vector should be used here as well.

I made a simple test by determining the distance between two points on either side of the plate boundary in Iceland. Using the cct calls below with and without the `+transpose` flag it is clear that without the the transpose flag the distance is getting smaller and with the flag the distance is increased. Since Iceland sits on a divergent plate boundary the distance should in fact increase. Ergo the `+transpose` flag is needed which means that the convention should be position vector.

```
cct -t2030 -z0 +proj=pipeline +ellps=GRS80 +step +proj=cart +step +init=ITRF2014:NOAM +t_epoch=2015.0 +transpose +step +proj=cart +inv

cct -t2030 -z0  +proj=pipeline +ellps=GRS80 +step +proj=cart +step +init=ITRF2014:EURA +t_epoch=2015.0 +transpose +step +proj=cart +inv
```

Reply:
OK I pushed 09bcf27 to fix that. Kind of scary to have to rely on such experimentations to guess the convention used. Isn't there raw data with the positions of the stations at various times ?

I had a look at https://agupubs.onlinelibrary.wiley.com/doi/pdf/10.1029/2011JB008930 and there's a note "[27]: The users of ITRF2008-PMM are advised to apply
not only plate angular velocities listed in Table 3, but also the translation rate components of Table 2." Should those translation rates be added ? Warning: I didn't read the paper (and couldn't make sense of it), just skimmed over it